### PR TITLE
finished task3, started task4

### PR DESCRIPTION
**Pull request by jlheig**

### CODE REVIEW

File: pokedex.xsl [hunk 1]
```diff
@@ -1,13 +1,21 @@
...
 	<!-- !!!!! Nous vous rendons attentif que vous n'avez PAS LE DROIT de modifier le reste du document, mais uniquement ces parties -->
 
-	</> <!-- ##### A compléter 1 -->
+	<xsl:output
+		method			= "html"
+		encoding		= "UTF-8"
+		doctype-public	= "-//W3c//DTD HTML 4.01//EN"
+		doctype-system	= "http://www.w3.org/TR/html4/strict.dtd"
+		indent 			= "yes"
+	/> <!-- ##### A compléter 1 -->
```
Comment (roosemberth) [inline pokedex.xsl:16]:
```suggestion
	/>
```

File: pokedex.xsl [hunk 2]
```diff
@@ -46,7 +54,7 @@
 
 					<div id="accordion">
 
-						<xsl:variable name="types" select="" /> <!-- ##### A compléter 3 : Ici, vous devez trouver l'expression XPath à mettre dans l'attribut select 
+						<xsl:variable name="types" select="distinct-values(//pokedex/pokemon/@type)" /> <!-- ##### A compléter 3 : Ici, vous devez trouver l'expression XPath à mettre dans l'attribut select
```
Comment (roosemberth) [inline pokedex.xsl:57]: Please remove the stray handout comment.

File: pokedex.xsl [hunk 3]
```diff
@@ -77,6 +85,9 @@
 								</xsl:attribute>
 
 								<!-- ##### A compléter 4 : Ici, vous devez faire appel au template lister_pokemon en lui passant le bon filtre en paramètre -->
```
Comment (roosemberth) [inline pokedex.xsl:87]:
```suggestion
```

File: pokedex.xsl [hunk 4]
```diff
@@ -110,7 +121,7 @@
 
 		</html>
 
-	<> <!-- Fin a compléter 2 -->
+	</xsl:template>> <!-- Fin a compléter 2 -->
```
Comment (roosemberth) [inline pokedex.xsl:124]:
```suggestion
	</xsl:template>>
```

Comment (roosemberth) [inline pokedex.xsl:124]:
```suggestion
	</xsl:template>
```

Closing tag seems to have an extra `>`. Please remove handout comments.

**Review by roosemberth — CHANGES_REQUESTED** [review]
Thanks for the patch, please mind the remarks attached to this review.

Please try to split each step in a different commit. Whilst it may not be necessary for this collaboration, it's better if you get yourself used to committing rather frequently. There's no need to reorganize the commits in this branch.

You may amend your commit and push-force or create and push a commit on top of this branch at your convenience.

**Comment by jlheig** [comment]
Decided to delete everything and start again from scartch because the stuff I did wasn't adequate

**Pull request closed without merging**